pkg/plugin/trans.go

```diff
@@ -273,6 +273,37 @@ func (r resourceTransition[T]) handleAutoscalingDisabled() (verdict string) {
 	return verdict
 }
 
+// handleStartMigration updates r.node with changes to clear any buffer and capacityPressure from
+// r.pod.
+//
+// If the pod is the migration source, this method *also* increases the node's PressureAccountedFor
+// to match the pod's resource usage.
+func (r resourceTransition[T]) handleStartMigration(source bool) (verdict string) {
+	// The first bit of this is _mostly_ the same as handleAutoscalingDisabled.
+
+	buffer := r.pod.Buffer
+	valuesToReduce := []*T{&r.node.Reserved, &r.node.Buffer, &r.pod.Reserved, &r.pod.Buffer}
+	for _, v := range valuesToReduce {
+		*v -= buffer
+	}
+
+	r.node.CapacityPressure -= r.pod.CapacityPressure
+	r.pod.CapacityPressure = 0
+
+	r.node.PressureAccountedFor += r.pod.Reserved
+
+	fmtString := "pod had buffer %d, capacityPressure %d; " +
+		"node reserved %d -> %d, capacityPressure %d -> %d, pressureAccountedFor %d -> %d"
```

Comment:
I'd just build this all as a giant `fmt.Sprint` call and avoid the string concatination (so you can use the buffer pool), and then you don't have to keep the deal with having the format string in the comment below as well.

Reply:
Leaving as-is for now because that's _currently_ the pattern used in the rest of the file.

(I'd imagine the compiler is ok doing this at compile-time though?)